theme/themes/pxt/globals/site.variables

```diff
@@ -20,6 +20,7 @@
 @greenTextColor: #107C10;
 
 @lightTextColor              : rgba(0, 0, 0, 0.68);
+@invertedUnselectedTextColor : rgba(255, 255, 255, 0.8); 
 
```

Comment:
This affects more than just the side docs. It affects inverted inputs as well. 
@transparentInvertedPlaceholderColor

Please add screenshots for all the components that are affected by the color change.